lib/ansible/modules/cloud/google/gcp_compute_address.py

```diff
@@ -101,11 +101,11 @@
 EXAMPLES = '''
 - name: create a address
   gcp_compute_address:
-      name: 'test-address1'
-      region: 'us-west1'
-      project: testProject
-      auth_kind: service_account
-      service_account_file: /tmp/auth.pem
+      name: test-address1
+      region: us-west1
+      project: "testProject"
```

Comment:
Would it be possible to have consistency around single quotes vs double quotes vs no quotes?

Reply:
A lot of this comes down to the Ruby YAML parser. It figures out when quotes are necessary and which ones it should use.